README.md

```diff
@@ -40,6 +37,38 @@ To see all the grunt tasks that are available:
 grunt help
 ```
 
+### Using Angular-PatternFly In Your Application
+
+Note:
+
+1. Add Angular-PatternFly as a dependency for your project and you'll receive all the libraries you'll need:
+
+        $ bower install angular-patternfly --save
+
+2. Add the follwing CSS includes to your HTML file(s), adjusting path where needed:
+
+        <!-- Angular-PatternFly Styles -->
+        <link rel="stylesheet" href="bower_components/angular-patternfly/dist/styles/angular-patternfly.min.css" />
+
+3. Add the following script includes to your HTML file(s), adjusting where necessary to pull in only what you need:
+
+        <!-- Angular-PatternFly  -->
+        <script src="bower_components/angular-patternfly/dist/angular-patternfly.min.js"></script>
+
+        <!-- C3, D3 - Charting Libraries -->
```

Comment:
I would put something here saying only required if you are using the patternfly.charts module